homeassistant/components/ted/translations/en.json

```diff
@@ -0,0 +1,36 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Device is already configured"
+        },
+        "error": {
+            "cannot_connect": "Failed to connect",
+            "unknown": "Unexpected error"
+        },
+        "flow_title": "{serial} ({host})",
+        "step": {
+            "user": {
+                "data": {
+                    "host": "Host"
+                }
+            }
+        }
+    },
+    "options": {
+        "step": {
+            "options": {
+                "data": {
+                    "spyder_energy_now": "Create Spyder Current Power entities",
+                    "spyder_energy_daily": "Create Spyder Daily Energy entities",
+                    "spyder_energy_mtd": "Create Spyder Month-to-Date Energy entities",
+                    "mtu_power_voltage": "Create MTU Current Voltage entities",
+                    "mtu_energy_now": "Create MTU Current Power entities",
+                    "mtu_energy_daily": "Create MTU Daily Energy entities",
+                    "mtu_energy_mtd": "Create MTU Month-to-Date Energy entities"
+                },
+                "description": "Configure TED integration",
+                "title": "TED options"
```

Comment:
This file is not in sync with `strings.json` file.
It also should not be modified manually, but only by a command line (don't remember which).

That said, I don't think options needs a description & title.

Reply:
Found it! `python3 -m script.translations develop`

Reply:
Thanks 😄